OloEngine/src/OloEngine/Renderer/Debug/FrameCaptureManager.cpp

```
@@ -0,0 +1,231 @@
+
+    void FrameCaptureManager::OnPostBatch(const CommandBucket& bucket)
+    {
+        if (!IsCapturing())
+            return;
+
+        m_PendingFrame.PostBatchCommands.clear();
+        DeepCopyCommands(bucket, m_PendingFrame.PostBatchCommands, true);
+        m_HasPendingPostBatch = true;
+    }
+
+    void FrameCaptureManager::OnFrameEnd(u32 frameNumber, f64 sortTimeMs, f64 batchTimeMs, f64 executeTimeMs)
```

Clang (14.0.6)

[warning] 4 adjacent parameters of 'OnFrameEnd' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)